### ISSUE TITLE
Hammer application logs on start

### ISSUE THREAD

**Issue by lpil** [ExHammer/hammer#18]
Hello!

The hammer application logs on start. In production this could possible be useful, but in dev and test it is unneeded noise that makes it harder to read test results and see deprecation warnings.

Would it be possible to remove this logging, or to make it configurable, or to only do it in the production mix env?

Thanks,
Louis

**Comment by ShaneKilkelly** [comment]
I'm sure it'd be easy to make this configurable, and that would actually be desirable in case we ever wanted to do more (optional) logging in Hammer.

Will make sure this gets into the next release :)

**Comment by lpil** [comment]
Great, thank you!

**Comment by ShaneKilkelly** [comment]
Fixed in commit: https://github.com/ExHammer/hammer/commit/8eaaab511dd7fd6b163f6e407c478d02472360f1

Will cut a release soon

**Comment by lpil** [comment]
Ace, thanks!

**Comment by ShaneKilkelly** [comment]
Released in `2.1.0`, use the `suppress_logs` config option:

```elixir
config :hammer,
  backend: {Hammer.Backend.ETS, []},
  suppress_logs: true
```
